docs/_documentations/troubleshooting.md

```diff
@@ -827,28 +945,21 @@ If you can run `docker` from the command line, but Codewind still fails to find
 3. Open the terminal you ran `docker` from and run `code`.
 4. Now, the new VS Code instance shares the terminal's `PATH`. Start Codewind again.
 
-<!--
-Action/Topic: Collecting log files and environment data
-Issue type: info
-Issue link: https://github.com/eclipse/codewind/issues/2766
-Info added in 0.12.0.
--->
-# Collecting log files and environment data
+## Build succeeded but project is stuck in the Stopped state
+If your project fails to start, check the application logs for anything that might indicate the failure.
+If your application does not have problems, you can [check the Codewind logs](troubleshooting.html#check-the-logs).
+If the error persists, you can reset the project by disabling and re-enabling the project.
 
-You can capture diagnostics from your installation by using the `cwctl diagnostics` CLI command to collect all available log files and environment information. You can find the `cwctl` CLI in your HOME directory under the `~/.codewind/<version>` path.  
+## No ESLint warnings or errors
+You see no ESLint warning or errors for Node.js projects. Install the [ESLint extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) and follow the instructions to activate the extension.
 
-The format of the command is: 
-`cwctl diagnostics [command options] [arguments...]`
+## Debug
 
-Command options are:
-* `--conid remote` - Triggers diagnostics collection for the remote codewind instance (_must_ have currently configured Kubectl connection, default:"local")
-* `--eclipseWorkspaceDir/-e <value>` - The location of your Eclipse workspace directory if using the Eclipse IDE, default:"")
-* `--intellijLogsDir/-i <value>` - The location of your IntelliJ logs directory if using the IntelliJ IDE (default: "")
-* `--quiet/-q` - Turn off console messages
-* `--projects/-p` - Collect project containers information
-* `--nozip/-n` - Does not create collection zip and leaves individual collected files in place
-* `--clean` - Removes the `diagnostics` directory and all its contents from the Codewind home directory
+### Liberty project remains in the Starting - Debug state
+Liberty servers started in **Debug** mode do not start until the debugger attaches. Run the `attach debugger` command] and the server starts. You can check the application logs to see if the server is starting.
```

Comment:
This is also general troubleshooting

Reply:
This is a very old Ts item for which I cannot find a corresponding issue. The remedial action -attach debugger- is covered in several places throughout the Ts, so removing.